components/inputs/input-number.js

```diff
@@ -337,6 +340,11 @@ class InputNumber extends FocusMixin(LabelledMixin(SkeletonMixin(FormElementMixi
 
 	render() {
 		const valueAlign = (this.valueAlign === 'end') ? 'end' : 'start';
+		const hasRelativeInputWidth = this.inputWidth.includes('%');
```

Comment:
I think `inputWidth` can be `undefined` in certain scenarios?

```suggestion
		const hasRelativeInputWidth = this.inputWidth?.includes('%');
```

Reply:
I don't think so, it's always defined in the constructor.